TwoPaneLayout/library/src/main/java/com/microsoft/device/display/twopanelayout/screenState/ScreenStateManager.kt

```diff
@@ -0,0 +1,91 @@
+package com.microsoft.device.display.twopanelayout.screenState
+
+import android.content.res.Configuration.ORIENTATION_PORTRAIT
+import android.graphics.Rect
+import android.os.Handler
+import android.os.Looper
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.platform.LocalView
+import androidx.core.util.Consumer
+import androidx.window.FoldingFeature
+import androidx.window.WindowLayoutInfo
+import androidx.window.WindowManager
+import java.util.concurrent.Executor
+
+const val SmallestTabletScreenWidthDp = 585
```

Comment:
A `const` should go in capital letters with underscores between words. (https://kotlinlang.org/docs/properties.html#compile-time-constants)